common/src/main/java/com/wynntils/overlays/minimap/MinimapOverlay.java

```diff
@@ -49,7 +49,7 @@ public class MinimapOverlay extends Overlay {
     private static final int DEFAULT_SIZE = 130;
 
     @Persisted
-    public final Config<Float> scale = new Config<>(1f);
```

Comment:
The upfixer is missing on purpose. Users will have to reset this value, as there is no 1 to 1 mapping between the new and old values (and rounding to new values is almost just as bad as a full reset).